lib/Thrift/Thrift.csproj

```diff
@@ -6,7 +6,8 @@
     <PackageIconUrl>https://opentelemetry.io/img/logos/opentelemetry-icon-color.png</PackageIconUrl>
     <PackageProjectUrl>https://OpenTelemetry.io</PackageProjectUrl>
     <PackageTags>Tracing;OpenTelemetry;Management;Monitoring;Jaeger;distributed-tracing</PackageTags>
-    <IsPackable>false</IsPackable>
```

Comment:
wait, why would you want to package this? this is an internal library that is compiled into Jaeger exporter.

Reply:
Here: 

https://github.com/open-telemetry/opentelemetry-dotnet/blob/9a7fde5719c445b9ca86c78f1d7db53e593ee01c/src/OpenTelemetry.Exporter.Jaeger/OpenTelemetry.Exporter.Jaeger.csproj#L13

Jaeger uses Thrift as a project reference, not nuget.